Sprint5 task4 admin add number of available kudos

iAcay opened the pull request:
Hey! The code which is below was implemented in order to meet the acceptance criterias of the last task in sprint no. 5.

AC in this task:
-- There's an "Add Kudos" link in the admin panel's navigation which opens a form for adding kudos
-- "Add Kudos" form has one numerical input for the number of kudos
-- The number of kudos in the "Add Kudos" form has to be between 1 and 20
-- Submitting the "Add Kudos" form adds the same number of kudos to the `number_of_available_kudos` field of each employee
-- After successfully submitting the "Add Kudos" form, admin sees a flash message with a confirmation
-- There's a system spec for admin adding kudos to multiple employees


https://user-images.githubusercontent.com/96024046/178150501-645fbcf1-9e57-4b57-b067-0b5786a4a1c4.mov

iAcay committed:
fix: change background color of alerts

iAcay committed:
fix: remove limitation for maximum number of available kudos

iAcay committed:
feat: possibility for admin to increase available kudos for all employees at once

iAcay committed:
refactor: put number_of_available_kudos into variable

iAcay committed:
tests: admin adds additional kudos for each employee at once

ciejj commented on app/controllers/AdminUsers/employees_controller.rb:
```diff
@@ -34,5 +58,9 @@ def employee
       params.require(:employee).permit(:email, :password, :number_of_available_kudos)
     end
+
+    def added_kudos_params
```
there is not real need to put that param into the method :) I would simply put it inside the line 37

iAcay replied:
First of all thanks a lot for CR :) I had followed your suggestion and removed this method :)

ciejj commented on app/controllers/AdminUsers/employees_controller.rb:
```diff
@@ -25,6 +25,30 @@ def destroy
+      render :edit_number_of_available_kudos_for_all, locals: { employees: Employee.all }
+    end
+
+    def update_number_of_available_kudos_for_all
```
since this is custom action I would consider naming it something like:
```suggestion
    def add_available_kudos_for_all
```

iAcay replied:
Changed :)

ciejj commented on app/controllers/AdminUsers/employees_controller.rb:
```diff
@@ -25,6 +25,30 @@ def destroy
       redirect_to admin_users_employees_path, notice: 'Employee was successfully destroyed.'
     end
 
+    def edit_number_of_available_kudos_for_all
```
do we really need that action? we do not even need the employees parameter - all you need is a form that allows to select a number with a submit button passing that value to the next action :)

iAcay replied:
No, we don't :D I have removed render action from this method ;)

ciejj reviewed (approved):
well done - left some ideas for minor improvements :)

iAcay committed:
fix: when entered filter is not reflecting any status

iAcay committed:
refactor: following CR suggestions

- remove render action from edit_number_of_available_kudos_for_all method
- remove added_kudos_param method
- change method name from update_number_of_available_kudos_for_all to add_available_kudos_for_all
